specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/interfaceEndpoint.json

```diff
@@ -0,0 +1,332 @@
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the interface endpoint."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/InterfaceEndpoint"
+            },
+            "description": "Parameters supplied to the create or update interface endpoint operation"
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Update successful. The operation returns the resulting InterfaceEndpoint resource.",
+            "schema": {
+              "$ref": "#/definitions/InterfaceEndpoint"
+            }
+          },
+          "201": {
+            "description": "Create successful. The operation returns the resulting InterfaceEndpoint resource.",
+            "schema": {
+              "$ref": "#/definitions/InterfaceEndpoint"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Create interface endpoint": { "$ref": "./examples/InterfaceEndpointCreate.json" }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/interfaceEndpoints": {
+      "get": {
+        "tags": [
+          "InterfaceEndpoints"
+        ],
+        "operationId": "InterfaceEndpoints_List",
+        "description": "Gets all interface endpoints in a resource group.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns a list of InterfaceEndpoint resources.",
+            "schema": {
+              "$ref": "#/definitions/InterfaceEndpointListResult"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List interface endpoints in resource group": { "$ref": "./examples/InterfaceEndpointList.json" }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/interfaceEndpoints": {
+      "get": {
+        "tags": [
+          "InterfaceEndpoints"
+        ],
+        "operationId": "ListInterfaceEndpointsInSubscription",

```

Comment:
The more idiomatic thing to do in other Network swaggers is to call this operation "ListAll". Also, not sure what is causing it here in the swagger, but in the generated Python SDK, it is appearing at the root of the network management client instead of under `interface_endpoints_operations`. @lmazuel ideas?

Reply:
@ravbhatnagar FYI

Reply:
kk, i'll rename it to that. I didn't realize there were guidelines on this :)

Reply:
No other subscription-based list commands do not appear in the base. They appear in the operation group they pertain to.

Reply:
Exactly, and the `name` before `_` in operation id becomes the operation group, which would be as expected per the latest change @rupalivohra made